astropy/modeling/utils.py

```diff
@@ -484,3 +484,59 @@ def combine_labels(left, right):
 
     return left + right
 
+
+def render_models2sky(model_list, image):
```

Comment:
I wasn't going to suggest that this add a different interface to `__call__`.  My thinking was that you would compute the x/y grids manually (which is easy using `np.mgrid` in conjunction with `model.bbox`, which I'm also proposing could return a slice that can be passed directly to `np.mgrid`.  I think it would be good to add a flag to `Model.__call__` to enable evaluation within the boundary as well.

However, I can also see the value in adding a higher-level shortcut to that, basically doing exactly what I described above (or, in other words, essentially what this function already does), but as a Model method (maybe `Model.render_to`), or `Model.rasterize_to`?  Maybe something like this (very rough sketch):

```python
def raster(self, to=None, copy=False):
    if to is None:
        img = np.zeros((self.bbox[0][1], self.bbox[1][1]))  # i.e. make an empty image
        # Alternatively, fill the background with NaNs?
    else:
        img = to  # Add to an existing image that was passed in
        # Also verify the size of the image and that the model is within its bounds
        if copy:
            img = img.copy()

    y, x = np.mgrid[...]
    output = self(x, y, bounded=True)

    img[self.bbox] += output
    # Optionally: Could support different blending modes as well as just add

    # if copy=False the image is modified in-place, but is still returned also, for consistency
    return img